java/yb-cql/src/test/java/org/yb/cql/SimpleQueryTest.java

```diff
@@ -0,0 +1,540 @@
+    }
+
+    assertRows(execute("SELECT * FROM %s"), expected);
+  }
+
+  @Test
+  public void test2ndaryIndexes() throws Throwable
+  {
+    createTable("CREATE TABLE %s (k text, t int, v text, PRIMARY KEY (k, t)) WITH transactions = { 'enabled' : true };");
+
+    createIndex("CREATE INDEX %s ON %s(v);");
+
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key1", 1, "foo");
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key1", 2, "bar");
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key2", 1, "foo");
+
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key2", 2, "foo");
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key2", 3, "bar");
+
+    assertRows(execute("SELECT * FROM %s WHERE v = ?", "foo"),
+      row("key1",  1, "foo"),
+      row("key2",  1, "foo"),
+      row("key2",  2, "foo")
+    );
+
+    assertRows(execute("SELECT * FROM %s WHERE v = ?", "bar"),
+      row("key1",  2, "bar"),
+      row("key2",  3, "bar")
+    );
+  }
+
+  @Test
+  public void testStaticColumns() throws Throwable
+  {
+    createTable("CREATE TABLE %s (k text, t int, s text static, v text, PRIMARY KEY (k, t));");
+
+    execute("INSERT INTO %s (k, t, v, s) values (?, ?, ?, ?)", "key1", 1, "foo1", "st1");
+    execute("INSERT INTO %s (k, t, v, s) values (?, ?, ?, ?)", "key1", 2, "foo2", "st2");
+
+    execute("INSERT INTO %s (k, t, v, s) values (?, ?, ?, ?)", "key1", 3, "foo3", "st3");
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key1", 4, "foo4");
+    execute("INSERT INTO %s (k, t, v, s) values (?, ?, ?, ?)", "key1", 2, "foo2", "st2-repeat");
+
+    execute("INSERT INTO %s (k, t, v, s) values (?, ?, ?, ?)", "key1", 5, "foo5", "st5");
+    execute("INSERT INTO %s (k, t, v) values (?, ?, ?)", "key1", 6, "foo6");
+
+
+    assertRows(execute("SELECT * FROM %s"),
+      row("key1",  1, "st5", "foo1"),
+      row("key1",  2, "st5", "foo2"),
+      row("key1",  3, "st5", "foo3"),
+      row("key1",  4, "st5", "foo4"),
+      row("key1",  5, "st5", "foo5"),
+      row("key1",  6, "st5", "foo6")
+    );
+
+    assertRows(execute("SELECT s FROM %s WHERE k = ?", "key1"),
+      row("st5"),
+      row("st5"),
+      row("st5"),
+      row("st5"),
+      row("st5"),
+      row("st5")
+    );
+
+    assertRows(execute("SELECT DISTINCT s FROM %s WHERE k = ?", "key1"),
+      row("st5")
+    );
+
+    assertEmpty(execute("SELECT * FROM %s WHERE k = ? AND t > ? AND t < ?", "key1", 7, 5));
+    assertEmpty(execute("SELECT * FROM %s WHERE k = ? AND t > ? AND t < ? ORDER BY t DESC", "key1", 7, 5));
+
+    assertRows(execute("SELECT * FROM %s WHERE k = ? AND t = ?", "key1", 2),
+      row("key1", 2, "st5", "foo2"));
+
+    displayRows(execute("SELECT * FROM %s WHERE k = ? AND t = ? ORDER BY t DESC", "key1", 2));
+    assertRows(execute("SELECT * FROM %s WHERE k = ? AND t = ? ORDER BY t DESC", "key1", 2),
+      row("key1", 2, "st5", "foo2"));
+  }
+
+  @Test
```

Comment:
Please file a Github issue on this test failure and add `@Ignore ("Disabled for issue #<issue number>")` to temporarily disable this test. We will continue to look into the issue.